### PR TITLE
chore: add pyment config

### PR DESCRIPTION
**Pull request by mikeldking**
This allows you to run pyment to generate documentation stubs using google format.

```sh
pyment -w -c ./pyment.conf ./src/phoenix/datasets/tracing.py
```

### CODE REVIEW

File: pyment.conf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+first_line = false
+quotes = '''
```
Comment (axiomofjoy) [inline pyment.conf:2]:
```suggestion
quotes = """
```

Comment (axiomofjoy) [reply pyment.conf:2]: For `ruff`.

**Review by axiomofjoy — APPROVED** [review]
Maybe make a `hatch` command?

File: pyment.conf (hunk 1 above)
Comment (mikeldking) [reply pyment.conf:2]: this is part of the pyment config. I'll fix

Comment (mikeldking) [reply pyment.conf:2]: oh nvm

**Comment by mikeldking** [comment]
> Maybe make a `hatch` command?

This will seed too much right now I think.

**Pull request merged**